video_xblock/backends/dummy.py

```diff
@@ -28,6 +28,7 @@ def media_id(self, href):  # pylint: disable=unused-argument
         """
         return '<media_id>'
 
+    @property
```

Comment:
How did this work at all?

Reply:
I think this place has never been reached before. Linter hinted me to change a type of this method.

Reply:
@sendr 

It works because `captions_api` is an `abstractproperty` in the base class.

Please review this - probably we can change it in bae class instead.

And also `abstractproperty` is deprecated.
```python
class abstractproperty(property):
    """A decorator indicating abstract properties.

    Deprecated, use 'property' with 'abstractmethod' instead.
    """

    __isabstractmethod__ = True
```